.github/workflows/pr-validate.yml

```diff
@@ -31,4 +35,10 @@ jobs:
       with:
         java-version: 1.8
     - name: mvn package
-      run: export LANG=en_US && ./mvnw -fae -B -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn clean package -Pcheck-format -DskipTests
+      run: |
+        ./mvnw -fae -B \
+          -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn \
+          -DskipTests \
+          -Pcheck-format \
+          clean package
```

Comment:
no problem to change it but since this is something outside the scope of this pr and as this behavior has not changed by this pr, I would change it with a follow up pr